direct/src/showbase/ShowBaseGlobal.py

```diff
@@ -11,10 +11,13 @@
 since ShowBase may not have been created yet; instead, ShowBase dynamically
 adds itself to this module's scope when instantiated."""
 
+from __future__ import annotations
+
 __all__ = ()
 
 from .ShowBase import ShowBase, WindowControls # pylint: disable=unused-import
 from direct.directnotify.DirectNotifyGlobal import directNotify, giveNotify # pylint: disable=unused-import
+from direct.directtools import DirectSession
```

Comment:
Not acceptable for ShowBaseGlobal to pull in the DIRECT tools.

Reply:
Hm. Would it be acceptable to get rid of that annotation, then? Or perhaps replace it with `Any`?

Alternatively, we could put the import under an `if typing.TYPE_CHECKING` guard.

Reply:
Let's not use a guard like that.  If there's no way to do it well without an import (or losing the typing benefit), I think we have to just move it out of ShowBaseGlobal into a new `directtools/DirectSessionGlobal.py` module.

Reply:
Does a string annotation not work in this case? If the situation is too funky, we may be better off leaving it un-annotated. I am not too concerned above covering every corner of direct, especially since it has some bad coding practices that can make annotations difficult.

Reply:
A string annotation is what I had before this change.  I assume there was a problem with that?

It doesn't really need to be in ShowBaseGlobal, and nobody relies on that since I added that only recently.  I think moving it into a new DirectSessionGlobal is possibly best if we can't do the annotation cleanly otherwise.

Reply:
Yeah, the string annotation didn't work because mypy had no way of knowing what `DirectSession` was. I'll just remove it.

Reply:
Okay, but then most of the code in DirectSession won't benefit from type checking.

Reply:
A DirectSessionGlobal module sounds like a good idea, but I think that's probably outside the scope of this PR.

Reply:
Fair.